iree/turbine/kernel/compiler/host_codegen.py

```diff
@@ -50,12 +51,11 @@ def isolated_test_call(
     dynamic_symbols: list[IndexSymbol] = [],
 ):
     with InsertionPoint(mb.body_block), Location.unknown():
-        input_types = [b.as_mlir_type() for b in sig.kernel_buffer_input_bindings]
+        input_types = [b.as_mlir_type() for b in sig.kernel_buffer_bindings]
         input_tensors = memref_to_tensor(input_types)
```

Comment:
Since now the input types have all the buffers do we need the code below for output types and output tensors?

Reply:
yeah, we still need it to set up the result types :)